src/test/java/com/wdtinc/mapbox_vector_tile/adapt/jts/MvtEncoderTest.java

```diff
@@ -86,8 +89,7 @@ public void multipleLayers() throws IOException {
         }
 
         private static Point createPoint(String name) {
-            Coordinate coord = new Coordinate( (int) (RANDOM.nextDouble() * 4095),
-                    (int) (RANDOM.nextDouble() * 4095));
+            Coordinate coord = new Coordinate(RANDOM.nextInt(4096), RANDOM.nextInt(4096));
```

Comment:
Same thing here, does this need to be changed as part of android compatibility?

Reply:
As before - a Findbugs warning for an unnecessary cast